static/css/style.css

```diff
@@ -1051,6 +1051,27 @@ p {
     font-size: 16px;
 }
 
+.answer_preview{
+    max-height: 500px;
+    overflow-y: scroll;
+    white-space: pre-wrap;

```

Comment:
This makes each paragraph have a large amount of spacing around it, which doesn't look good IMO.

Reply:
I'm talking about line 1057 in particular. I don't know why it highlighted all those other lines.

Reply:
You're right, I removed the line.